app/class_room.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :lable
+
+  @students = []
```

Comment:
In order to allow getting the students belonging to a class, kindly specify a getter method. You may use the attr_reader shortcut as thus: `attr_reader :students`